docs/organizations/managing-people.md

```diff
@@ -28,44 +30,44 @@ To join or add an organization after completing the sign-up process, click **Org
 
 ## Adding people to your organization {: id="adding-people"}
 
-**On Codacy Cloud**, organization owners can also add team members to their organization on Codacy. This is useful to allow Codacy to analyze commits to private repositories by contributors who haven't signed up to Codacy or joined the organization yet.
+**On Codacy Cloud**, organization owners can also add teammates to their organization on Codacy. This is useful to allow Codacy to analyze commits to private repositories by committers who haven't signed up to Codacy or joined the organization yet.
 
 !!! tip
     You can also use the Codacy API to [add people to your Codacy organization](../codacy-api/examples/adding-people-to-codacy-programmatically.md). This is useful while adding a large amount of people or to automatically add new members of your Git provider organization to Codacy.
 
 !!! note
-    For Enterprise plans where Codacy automatically adds new people that commit to your private repositories, you can invite them to join your organization on the Codacy app.
+    For Enterprise plans where Codacy automatically adds new committers to your organization, you can invite them to join the organization on the Codacy app.
 
-To add members to your organization:
+To add people to your organization:
 
 1.  Open your organization **Settings**, page **People**, and click the button **Add people**.
 
-    ![Adding members to your organization](images/organization-people-add-button.png)
+    ![Adding people to your organization](images/organization-people-add-button.png)
 
-1.  Select people from the list displaying pending requests to join the organization, as well as recent contributors to the private repositories in your organization.
+1.  Select people from the list displaying pending requests to join the organization, as well as recent committers to the private repositories in your organization.
 
     Alternatively, click **Add people using email addresses** to manually enter the list of email addresses of the people you wish to add.
 
-    ![Adding members to your organization](images/organization-people-add-modal.png)
+    ![Adding people to your organization](images/organization-people-add-modal.png)
 
-1.  Confirm the updated billing details displayed at the bottom of the window and click the button **Add people**. Codacy emails the newly added members inviting them to log in.
+1.  Confirm the updated billing details displayed at the bottom of the window and click the button **Add people**. Codacy emails the newly added people inviting them to log in.
 
 ## Removing people from your organization {: id="removing-people"}
 
-Members of an organization on Codacy can remove themselves from the organization, and organization owners can also remove other members.
+Members of an organization on Codacy can remove themselves from the organization, and organization owners can also remove other members or committers.
```

Comment:
I think that here an "and" is more correct since organization owners can remove members **and** committers.

```suggestion
Members of an organization on Codacy can remove themselves from the organization, and organization owners can also remove other members and committers.
```